beacon-chain/core/blocks/attester_slashing.go

```diff
@@ -152,7 +152,7 @@ func IsSlashableAttestationData(data1, data2 *ethpb.AttestationData) bool {
 	return isDoubleVote || isSurroundVote
 }
 
-func slashableAttesterIndices(slashing *ethpb.AttesterSlashing) []uint64 {
+func SlashableAttesterIndices(slashing *ethpb.AttesterSlashing) []uint64 {
```

Comment:
When making a function public, please add a godoc commentary.

```suggestion
// SlashableAttesterIndices returns the intersection of attester indices from both attestations in this slashing.
func SlashableAttesterIndices(slashing *ethpb.AttesterSlashing) []uint64 {
```